gemfiles/RailsHeadGemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+gem "rails", github: "rails/rails", branch: "master", require: false
+gem 'rubocop', '~> 0.56.0'
```

Comment:
Yes, that was initially my first approach to adding those gems, but then I figured that since this is a gemfile that is going to run only on Travis CI this is going to be used only on a test environment so it wouldn't make a difference to specify that. If you think I should still add them in a different group please let me know.

Reply:
Yes, I understand. But I guess it's better to keep things the right way. It seems this is what `devise` does: https://github.com/plataformatec/devise/blob/master/gemfiles/Gemfile.rails-4.1-stable